src/Components/Facility/AssetCreate.tsx

```diff
@@ -480,40 +480,42 @@ const AssetCreate = (props: AssetProps) => {
                   <div className="col-span-6 flex flex-col lg:flex-row gap-x-12 xl:gap-x-16 transition-all">
                     {/* Location */}
                     <div>
-                      <label htmlFor="asset-location">Location * </label>
+                      <label htmlFor="asset-location">Location *</label>
                       <div className="mt-2">
-                        <SelectMenu
+                        <SelectMenuV2
+                          required
                           options={[
                             {
                               title: "Select",
-                              description:
-                                "Select an Asset Location from the following",
-                              value: "",
+                              description: "Select the location",
+                              value: "0",
                             },
                             ...locations.map((location: any) => ({
                               title: location.name,
                               description: location.facility.name,
                               value: location.id,
                             })),
                           ]}
-                          selected={location}
-                          onSelect={setLocation}
+                          optionLabel={(o) => o.title}
+                          optionValue={(o) => o.value}
+                          value={location}
+                          onChange={(e) => setLocation(e)}
                         />
                       </div>
                       <ErrorHelperText error={state.errors.location} />
                     </div>
 
                     {/* Asset Type */}
                     <div>
-                      <label htmlFor="asset-type">Asset Type * </label>
+                      <label htmlFor="asset-type">Asset Type *</label>
                       <div className="mt-2">
-                        <SelectMenu
+                        <SelectMenuV2
+                          required
                           options={[
                             {
                               title: "Select",
-                              description:
-                                "Select an Asset Type from the following",
-                              value: undefined,
+                              description: "Select the asset type",
+                              value: "NONE",
                             },
```

Comment:
Remove the default option and use the placeholder property to achieve the same instead.